chrome/views/popupView.html

```diff
@@ -9,54 +9,57 @@
         <a id="close-button" title="Close">&times;</a>
     </div>
     <div class="popupMsg">Loading...</div>
-    <div class="content" style="display: none;">
-        <div class="gttTopRight">
-            <div style="float: left;" class="clearfix listrow">
-                <ul id="gttPosition" />
+    <div class="outerContent">
```

Comment:
Had to create an outerContent div and indent everything inside it, so I can scroll the popup when all the Trello cards make it too long to fit on the screen.